src/tests/Microsoft.PowerFx.Core.Tests/ThreadingTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    // Do static analysis to look for potential threading issues. 
+    public class ThreadingTests
+    {
+        [Fact]
+        public void CheckFxCore()
+        {
+            var asm = typeof(Core.Syntax.Nodes.TexlNode).Assembly;
+
+            var bugsFieldType = new HashSet<Type>
```

Comment:
>bugsFieldType [](http://example.com/codeflow?start=16&length=13)

Bug list here captures work on Fx Core to get it "clean". 
Will handle these in followup PRs.